package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -464,44 +483,39 @@ public static GeometrySubsets ReadGeomSubsets(Scene scene, string path)
                 }
                 else
                 {
+                    var usdIndices = usdMesh.faceVertexIndices;
                     unityMesh.subMeshCount = geomSubsets.Subsets.Count;
-                    int subsetIndex = 0;
+                    var subsetIndex = 0;
                     foreach (var kvp in geomSubsets.Subsets)
                     {
-                        int[] faceIndices = kvp.Value;
-                        int[] triangleIndices = new int[faceIndices.Length * 3];
+                        var faceIndices = kvp.Value;
```

Comment:
Changing this to `var` makes the code harder to read, because we also don't know what types `kvp` holds.

Reply:
Added in dd63d81